src/test/java/com/microsoft/jenkins/kubernetes/integration/TestHelpers.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.jenkins.kubernetes.integration;
+
+import hudson.FilePath;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.UUID;
+
+public class TestHelpers {
```

Comment:
Move some common methods to common plugin so that they can be reused?

Reply:
make sense. I will copy the logic to commons and when that's available in new commons release, we can remove the duplicates in the descendants.